private/solr-conf/synonyms_und.txt

```diff
@@ -11,35 +11,8 @@
 
 phone,telephone
 tv,television
-911 call,911 calls,dispatch call,dispatch calls,emergency call,emergency calls
-address,addresses,location,locations
 adu,accessory dwelling unit
-appeal,appeals,petition,petitions
-building permit,building permits,construction permit,construction permits
-business license,business registration,business permit
-business search=>business lookup
-certificate of compliance,certificate of conformance,certificate of conformity
-classes,lessons,instructions,training
-directory,catalogue,roster
-employee,staff,personnel
 garbage,rubbish,waste,trash,debris,litter
-gym,gymns,gymnasium,gynmnasiums,health club,health clubs,fitness club,fitness clubs,fitness center,fitness centers
-internship,internsips,apprenticeship,apprenticeships,traineeship,traineeships,work experience
-job,jobs,career,careers,employment
```

Comment:
I'd recommend keeping this one since jobs are our #1 search.

Reply:
Yes, but the search logs only ever show folks search for job or jobs, which are stems and don't need a synonym. Career doesn't get enough work to count at the moment.